deploy/clowdapp.yaml

```diff
@@ -43,7 +43,7 @@ objects:
                 value: ${LOGGING_LEVEL}
               - name: REST_ENDPOINTS_TRACE_DATA
                 value: ${REST_ENDPOINTS_TRACE_DATA}
-              - name: DATABASE_LOGGING_LEVEL
+              - name: DATABASE_LOG_LEVEL
```

Comment:
You've probably missed my comment:

Now the clowder parameter is named differently then the env value, do you think it would make sense to name them the same?
Renaming the paramter to LOGGING might be bit easier (just one line PR and two line MR to app-interface), right?

Reply:
Right, okay, I am taking 180 deg turn, how about this?